test/test_support.ts

```diff
@@ -19,6 +19,21 @@ import {toClosureJS} from '../src/main';
 import {sourceMapTextToConsumer} from '../src/source_map_utils';
 import * as tsickle from '../src/tsickle';
 
+/**
+ * The BASE_PATH that test source file paths are relative to.
+ *
+ * This is the path that contains the 'test_files' or 'test' folder. This can be the bazel runfiles
+ * directory (i.e. the workspace root), or potentially a subdirectory if the tsickle source
+ * code exists in a subdirectory in a different bazel workspace.
+ */
+const BASE_PATH = (() => {
+  const runfiles = process.env['RUNFILES'];
+  const candidate =
+      path.join(runfiles, 'google3', 'third_party', 'javascript', 'node_modules', 'tsickle');
```

Comment:
FWIW google3 will never exist in a non-Unix form, so you might as well write the second arg to this function
'google3/third_party/javascript/node_modules/tsickle'
as that will be easier to grep for (e.g. if we ever move it to a versioned subdir)

Reply:
Yeah, done.